### PR TITLE
mid.providerfacility translation

### PR DESCRIPTION
**Pull request by carresemata**
Actions in this PR:
- translate mid.providerfacility
- add view needed for the translation called base.vwupdcfacilitydetail
- solve table dependencies errors that X-ray identified

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Mid/ProviderFacility/spu_translated_ProviderFacility.sql [hunk 1]
```diff
@@ -0,0 +1,829 @@
...
+    fta.mergeddata as mrgd,
+    fta.isbestind as isbest,
+    fta.ismaxyear as ismaxyr
+from
+    ermart1.facility_facilitytoaward fta
+    join base.award award on fta.awardname = award.awardname
+    join base.awardcategory award_cat on award_cat.awardcategoryid = award.awardcategoryid
+    left join ermart1.facility_serviceline svc_line on fta.specialtycode = svc_line.servicelineid
+group by
+    fta.facilityid,
```
Comment (ggandia-xtillion) [inline migration_original/ODS1Stage/tables/Mid/ProviderFacility/spu_translated_ProviderFacility.sql:412]: remove columns not in select statement

File: migration_original/ODS1Stage/tables/Mid/ProviderFacility/spu_translated_ProviderFacility.sql [hunk 2]
```diff
@@ -0,0 +1,829 @@
...
+    tpsl.providerid,
+    tqs.facilityid,
+    tqs.servicelinecode,
+    tqs.ratingssortvalue
+from
+    cte_tempqualityscoretable tqs
+    join cte_tempproviderspecialtyserviceline tpsl on tqs.servicelinecode = tpsl.servicelinecode
+    join cte_pfacility pf on tqs.facilityid = pf.legacykey and tpsl.providerid = pf.providerid
+where
+    iff(tqs.ratingssortvalue is null, -1, tqs.ratingssortvalue) <> -1
```
Comment (ggandia-xtillion) [inline migration_original/ODS1Stage/tables/Mid/ProviderFacility/spu_translated_ProviderFacility.sql:527]: ifnull

File: migration_original/ODS1Stage/tables/Mid/ProviderFacility/spu_translated_ProviderFacility.sql [hunk 3]
```diff
@@ -0,0 +1,829 @@
...
+    )::varchar
+    ,
+    '',
+    'phone') as pdcphonexml
+from cte_phonepdc
+group by
+    clientproducttoentityid
+),
+cte_providerfacility as (
+select distinct
```
Comment (ggandia-xtillion) [inline migration_original/ODS1Stage/tables/Mid/ProviderFacility/spu_translated_ProviderFacility.sql:578]: remove distinct

**Review by ggandia-xtillion — CHANGES_REQUESTED** [review]

**Comment by carresemata** [comment]
Added all changes

**Review by ggandia-xtillion — APPROVED** [review]
Screenshot_2024-04-04_at_10 27 52_AM-removebg-preview (1)

**Pull request merged**